styla.php

```diff
@@ -252,9 +252,9 @@ private function buildContentKey() {
             $contentKey = isset($_REQUEST["origUrl"]) ? $_REQUEST["origUrl"] : "/";
 
             if($this->rootPath !== "/"){
-                $path = join([ "/", $this->rootPath ]);
```

Comment:
@ssachtleben is the `rootPath` param always a string? I am assuming so for the fix.

Reply:
yes